pom.xml

```diff
@@ -352,6 +353,13 @@
                 <scope>test</scope>
             </dependency>
 
+            <dependency>
+                <groupId>com.dameng</groupId>
+                <artifactId>DmJdbcDriver18</artifactId>
+                <version>${dm-jdbc.version}</version>
+                <scope>provided</scope>
```

Comment:
this `scope` need to be `test`. We don't want to include DmJdbcDriver18.jar in the connector's package